assets/javascripts/needleeditor.js

```diff
@@ -570,5 +610,12 @@ function setup_needle_editor(imageurl, default_needle)
   });
 }
 
+function initWorkaroundDesc(enabled) {
```

Comment:
This function is not initializing (initializing means a first-time assignment of initial values). It just updates the visibility. Before thinking about a better name I'd suggest to inline that function. It is only called once and the calling function is so far only 2 lines.

Reply:
your suggestion is good. I will fix it.